jacobian_follower/src/jacobian_follower/jacobian_follower.cpp

```diff
@@ -294,7 +297,7 @@ std::optional<moveit_msgs::RobotState> JacobianFollower::computeCollisionFreePoi
   bool ok = false;
   auto attempts{0};
   for (; attempts < max_collision_check_attempts and not ok; ++attempts) {
-    robot_state_ik.setToRandomPositions(joint_model_group);
+    robot_state_ik.setToRandomPositionsNearBy(joint_model_group, seed_robot_state_ik, 0.1);
```

Comment:
better to define a constant (file level would be fine) than have in-line magic numbers